src/main/java/seedu/address/model/Model.java

```diff
@@ -84,4 +89,9 @@ public interface Model {
      * @throws NullPointerException if {@code predicate} is null.
      */
     void updateFilteredPersonList(Predicate<Person> predicate);
+
+    /**
+     * Returns an unmodifiable view of the filtered event list
+     */
+    ObservableList<Event> getFilteredEventList();
```

Comment:
```suggestion
    /** Returns an unmodifiable view of the filtered event list */
    ObservableList<Event> getFilteredEventList();
```

Standardization with `getFilteredPersonList` method

Reply:
Maybe should standardise the other way around, make `getFilteredPersonList` method javadoc standardised with this more typical format thats used for eg `deleteEvent` javadoc

Reply:
Ok, it might be /** */ if its fits within a single line and 
/** 
*
*/
if the comment takes up more that a line so I'll edit to the first suggestion